remotes/docker/pusher.go

```diff
@@ -105,6 +134,7 @@ func (p dockerPusher) Push(ctx context.Context, desc ocispec.Descriptor) (conten
 
 			if exists {
 				p.tracker.SetStatus(ref, Status{
+					Committed: true,
```

Comment:
Should the `Offset`/`Total` be reset here too?

(also applies to the other occurrence around like 195)

Reply:
Added Offset/Total here too.